### DISCUSSION TITLE
Documentation Update Proposal (ui.colors)

### DISCUSSION THREAD

**Discussion by Xtreemrus** [zauberzeug/nicegui#3180] [Ideas / Feature Requests]
Hello,
In the [NiceGUI documentation ui.colors](https://nicegui.io/documentation/colors), there is a link to Quasar under the text "used by Quasar". Currently, this link directs users to the main page of [Quasar](https://quasar.dev/).

Propose updating this link to point directly to the Quasar Theme Builder page at https://quasar.dev/style/theme-builder. This change will provide users with immediate access to a more specific and useful resource where they can directly view and experiment with colors and themes, especially for primary, secondary, and accent variables.

Benefits
Improved User Experience: Users can quickly access the theme builder, which is directly relevant to their needs.
Enhanced Resource Utilization: Direct access to the theme builder page allows users to see and customize themes and colors more efficiently.

Maybe this might also help us see more projects using NiceGUI with different themes than the standard blue one :)

**Comment by falkoschindler** [comment]
Great idea, @Xtreemrus!
I quickly changed it in 3a1448010ff3511cbd8823a3fca83ac4a967597a. 🚀

> Maybe this might also help us see more projects using NiceGUI with different themes than the standard blue one :)

Oh yes! 😀
